festflow/static/css/contact.css

```diff
@@ -181,4 +182,758 @@ figure.effect-duke:hover p {
 	}
 }
 
+/*!
```

Comment:
ok not entire but a lot of. You could have simply used bootstrap as a separate include.